apps/relgen/src/index.ts

```diff
@@ -376,11 +376,11 @@ release
     let prompt: string | undefined;
 
     if (templateFile) {
-      template = await readFile(templateFile, 'utf-8');
+      template = (await readFile(templateFile, 'utf-8')).trim();
```

Comment:
The use of 'trim' is not allowed as per the rules. Please find an alternative method to handle whitespace.